database/migrations/2017_09_20_095030_create_endpoints_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateEndpointsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('endpoints', function (Blueprint $table) {
+            $table->increments('id');
+
+            $table->string('method', 7);
```

Comment:
Sweet! 7 characters is indeed all we need. 👌 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Reply:
Thanks 👍🏻